proposed/extended-coding-style-guide.md

```diff
@@ -247,6 +247,89 @@ class ClassName extends ParentClass implements
 }
 ```
 
+### Using traits
+
+The `use` keyword used inside the classes to implement traits MUST be
+declared one line after the classe's opening brace, and MUST be indented
+with 4 space.
+
+```
+<?php
+namespace Vendor\Package;
+
+use Vendor\Package\FirstTrait;
+
+class ClassName
+{
+    use FirstTrait;
+}
+```
+
+When the `use` declaration exceeded 120 lines, use muiltiple lines for
```

Comment:
I'm not sure how much we should make a judgement on line length at all. The length requirements as written pretty much say  "Lines MAY be any length and MUST NOT be limited." with some extra wording around how and when analysis tools warn you. It's really a bunch of "SHOULD"s with the only requirement being that compliant code bases "MUST" not have a hard limit.  
Other than that, it reads more like it's directed at code analysis tools and not at code style compliance, and I think we should stay away from that.